collect_app/src/test/java/org/odk/collect/android/instancemanagement/autosend/InstanceAutoSendFetcherTest.kt

```diff
@@ -0,0 +1,82 @@
+package org.odk.collect.android.instancemanagement.autosend
+
+import org.hamcrest.MatcherAssert.assertThat
+import org.hamcrest.core.Is.`is`
+import org.junit.Assert.assertTrue
+import org.junit.Before
+import org.junit.Test
+import org.mockito.kotlin.mock
+import org.mockito.kotlin.whenever
+import org.odk.collect.forms.instances.Instance
+import org.odk.collect.formstest.FormUtils.buildForm
+import org.odk.collect.formstest.InMemFormsRepository
+import org.odk.collect.formstest.InMemInstancesRepository
+import org.odk.collect.formstest.InstanceUtils.buildInstance
+import org.odk.collect.projects.Project
+import org.odk.collect.shared.TempFiles.createTempDir
+
+class InstanceAutoSendFetcherTest {
+    private lateinit var instanceAutoSendFetcher: InstanceAutoSendFetcher
```

Comment:
I think you can just assign the field here and avoid the `lateinit` right?

Reply:
Right, done.